WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -253,6 +253,18 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Login Epilogue, in the specified NavigationController.
     ///
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void) {
+        let matcher = ULAccountMatcher()
+
+        guard let siteURL = credentials.wpcom?.siteURL, matcher.match(originalURL: siteURL) else {
+            //fatalError("Self Hosted sites are not supported. Please review the Authenticator settings!")
+            DDLogWarn("⚠️ Present account mismatch error for site: \(String(describing: credentials.wpcom?.siteURL))")
+            let viewModel = WrongAccountErrorViewModel(siteURL: credentials.wpcom?.siteURL)
+            let mismatchAccountUI = ULAccountMismatchViewController(viewModel: viewModel)
+            navigationController.pushViewController(mismatchAccountUI, animated: true)
+
+            return
```

Comment:
Nit: This could be reduced into:
```suggestion
            return navigationController.show(mismatchAccountUI, sender: nil)
```